recipe/run_test.py

```diff
@@ -382,21 +270,11 @@ def check_libraries(libraries_files):
     check_outputs(nnet2_bins)
     check_outputs(nnet3_bins)
     check_outputs(rnnlm_bins)
+    check_outputs(online2_bins)
 
     if test_cuda:
         check_outputs(cuda_decoder_bins)
         check_outputs(cuda_feat_bins)
 
     if sys.platform != 'win32':
         check_outputs(online_bins)
-        check_outputs(online2_bins)
-
-    print("Currently skipping header checks!")
-    for k, v in headers.items():
-        continue
-        if sys.platform == 'win32' and k in ['online', 'online2']:
-            continue
-        check_headers(k, v)
-
-    if sys.platform != 'win32':
-        check_libraries(libraries)
```

Comment:
Missing newline

Reply:
Still missing.

Reply:
Weird, it was definitely there on my machine, but I've added an extra newline so now there's 2 on my machine and one on github...

Reply:
Yeah, sorry that was my bad. It's still showing in the diff because it was missing _before_ this PR.